app/main.py

```diff
@@ -1 +1,64 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+
+        if self.is_hungry is False:
+            self.appetite = 0
+            return 0
```

Comment:
No need to change appetite to 0
Just return 0
```suggestion
       return 0
```

Reply:
When I make this changes, my tests fail